src/mongocxx/include/mongocxx/v_noabi/mongocxx/exception/error_code.hpp

```diff
@@ -107,6 +107,9 @@ enum class error_code : std::int32_t {
     // A default-constructed or moved-from mongocxx::v_noabi::search_index_view object has been
     // used.
     k_invalid_search_index_view,
+    
+    // Timed out while waiting for a client to be returned to the pool
+    k_pool_wait_queue_timeout,
```

Comment:
```suggestion

    // Timed out while waiting for a client to be returned to the pool
    k_pool_wait_queue_timeout,
```
Formatting (remove trailing whitespace).